docs/source/routing/self-hosted/containerization/aws.mdx

```diff
@@ -0,0 +1,162 @@
+
+    ```bash showLineNumbers=false
+    docker tag router-aws:v1.61.0 0123456789000.dkr.ecr.us-west-1.amazonaws.com/router-repo:v1.61.0
+    ```
+
+1. Run `docker push` to push the router image to your ECR repository URI, using a tag (e.g., `:v1.61.0`):
+
+    ```bash showLineNumbers=false
+    docker push 0123456789000.dkr.ecr.us-west-1.amazonaws.com/router-repo:v1.61.0
+    ```
+
+1. Run `aws ecr list-images` and validate that your image is in the list of images in your ECR repository:
+
+    ```bash showLineNumbers=false
+    aws ecr list-images --repository-name router-repo
+    ```
+
+## Create and deploy ECS task
+
+With your image pushed to your ECR repository, in ECS you can define a task for the router and deploy it as a service.
+
+### Create cluster
+
+You need an ECS cluster to deploy the router.
+
+If you don't have a cluster, you can create one with default settings: 
+
+1. In the AWS Console, go to the Amazon ECS Console, then click **Create cluster**.
+1. Enter a name for your cluster.
+1. Click **Create**.
+
+### Create task definition
+
+Create an ECS task definition for your router:
+
+1. In the AWS ECS Console, go to **Task definitions** from the left navigation panel, then click **Create new task definition** and select **Create new task definition**.
+1. Fill in the details for **Container - 1**:
+    - **Name**: Enter a container name
+    - **Image URI**: Select the URI of your router image
+    - **Port mappings**:  
+        - **Container port**: Enter `4000` (must match ) and 
+        - **Port name**: Enter a port name
+    - **Environment variables**: Enter the environment variables `APOLLO_KEY` and `APOLLO_GRAPH_REF` and set them to your graph API key and graph ref, respectively
+1. In `Docker configuration - optional`, enter the command options to configure the router and run it in development mode:
+
+    ```text showLineNumbers=false
+    --dev, --config, /dist/config/router.yaml
+    ```
+1. Click **Create**. 
+
+### Deploy router
+
+Deploy the router in your ECS cluster:
+
+1. In AWS ECS Console under **Task definitions**, select your defined task, then click **Deploy** and select **Create service**.
+1. Fill in  the fields for the service:
+    - **Existing cluster**: Select your cluster
+    - **Service name**: Enter a name for your service
+1. Click **Create** to create the service. ECS will start deploying the service for the router.
+1. After AWS finishes deploying, click on the service to go to its page in Console. Check the service logs for messages from the running router. For example:
+
+    ```text title="Example router log message"
+    {"timestamp":"2025-04-04T17:32:14.928608731Z","level":"INFO","message":"Apollo Router v1.61.0 // (c) Apollo Graph, Inc. // Licensed as ELv2 (https://go.apollo.dev/elv2)","target":"apollo_router::executable","resource":{}}
+    ```
+1. Go to the service URL and validate the the router's development Sandbox is running successfully.
```

Comment:
```suggestion
1. Go to the service URL and validate the router's development Sandbox is running successfully.
```

Is it worth showing a screenshot or otherwise indicating what a successfully running Sandbox looks like?